package.json

```diff
@@ -26,13 +26,18 @@
     "@typescript-eslint/eslint-plugin": "^2.28.0",
     "@typescript-eslint/parser": "^2.28.0",
     "babel-loader": "^8.0.4",
+    "css-loader": "^3.5.2",
     "eslint": "^6.8.0",
     "eslint-config-airbnb": "^18.1.0",
     "eslint-plugin-import": "^2.20.2",
     "eslint-plugin-jsx-a11y": "^6.2.3",
     "eslint-plugin-react": "^7.19.0",
     "eslint-plugin-react-hooks": "^2.5.1",
     "html-webpack-plugin": "^3.2.0",
+    "mini-css-extract-plugin": "^0.9.0",
+    "node-sass": "^4.13.1",
+    "sass-loader": "^8.0.2",
+    "style-loader": "^1.1.4",
```

Comment:
@blaise82 just curious, among those packages above don't we need saas package too?

Reply:
which sass pachage?

Reply:
- node-sass provides binding for Node.js to LibSass, a Sass compiler.
- sass-loader is a loader for Webpack for compiling SCSS/Sass files.
- style-loader injects our styles into our DOM.
- css-loader interprets @import and @url() and resolves them.
- mini-css-extract-plugin extracts our CSS out of the JavaScript bundle into a separate file, essential for production builds.